cypress/support/pages/home.pageObject.js

```diff
@@ -1,10 +1,38 @@
 import PageObject from '../PageObject';
 
 class HomePageObject extends PageObject {
-  url = '/#/';
+  url = '#/';
 
   get usernameLink() {
-    return cy.getByDataCy('username-link');
+    return cy.getByDataQa('username-link');
+  }
+
+  clickUsernameLink() {
+    this.usernameLink
+      .click();
+  }
+
+  assetrNewUsername(newUsername) {
```

Comment:
There's a typo in the method name. It should be 'assertNewUsername', not 'assetrNewUsername'.